### PR TITLE
A0-2762: Remove non-A0 networks support

### PR DESCRIPTION
**Pull request by jalooc**
Only keeping the A0 networks as per the discussion linked in the ticket:
image

Did a clean up of all the unusued networks resources as well - that should reduce the app bundle size and speed up IDEs indexing.

### CODE REVIEW

File: packages/apps-config/src/api/index.ts [hunk 1]
```diff
@@ -1,10 +1,7 @@
...
 // SPDX-License-Identifier: Apache-2.0
 
-import { applyDerives } from './derives.js';
-import { typesBundle as typesOnlyBundle } from './typesBundle.js';
+export { typesBundle } from './typesBundle.js';
 
 export * from './params/index.js';
 export * from './constants.js';
-
-export const typesBundle = applyDerives(typesOnlyBundle);
```
Comment (jalooc) [inline packages/apps-config/src/api/index.ts:10]: From what I understand, derives relate only to [some chains](https://github.com/Cardinal-Cryptography/apps/pull/74/files#diff-a5facc6227c8d5c5461d1550e9112093a18f9e94be4d74bd5fda8439bdfe0416L13-L17) - not to A0.

**Review by kostekIV — COMMENTED** [review]
Is the A0-2726 correct ticket for this pr?

**Comment by jalooc** [comment]
It's actually A0-2762 - thanks for noticing.

**Review by kostekIV — APPROVED** [review]
lgtm

**Review by Marcin-Radecki — APPROVED** [review]

**Pull request merged**